### PR TITLE
OCPBUGS-25933: Add policy audit maxLogFiles

### PR DESCRIPTION
**Pull request by jab-rh**
- https://issues.redhat.com/browse/OCPBUGS-25933

Version(s): 4.13+

Issue:

Link to docs preview: [Audit configuration](https://72754--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/logging-network-policy#network-policy-audit-configuration-logging-network-policy)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@asood-rh, can you confirm? Thanks!

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -296,6 +296,10 @@ This field cannot be changed after installation.
 |integer
 |The maximum size for the audit log in bytes. The default value is `50000000` or 50 MB.
 
+|`maxLogFiles`
+|integer
+|The maximum number of log files the audit log.
```
Comment (asood-rh) [inline modules/nw-operator-cr.adoc:301]: @jab-rh Could we further clarify that these are log files that are retained or at a given time?
Something like this 
The maximum number of  ACL audit log files retained.

Comment (jab-rh) [reply modules/nw-operator-cr.adoc]: @asood-rh, I've updated this.

**Comment by obrown1205** [comment]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

LGTM

**Pull request merged**